packages/fungible-token/bridge-fungible-token/src/main.sw

```diff
@@ -257,3 +263,8 @@ fn _asset_to_sub_id(asset_id: AssetId) -> b256 {
     require(sub_id.is_some(), BridgeFungibleTokenError::AssetNotFound);
     sub_id.unwrap()
 }
+
+#[storage(read)]
+fn _check_asset_is_minted_by_contract(asset_id: AssetId) {
+    require(storage.asset_to_sub_id.get(asset_id).try_read().is_some(), BridgeFungibleTokenError::AssetNotFound);
+}
```

Comment:
I would remove this because it's only used twice and it has a single require. If it did a bit more then I would potentially keep it

Reply:
Removed